r/tests/testthat/test-dataset.R

```diff
@@ -97,6 +97,37 @@ test_that("Simple interface for datasets", {
   )
 })
 
+test_that("Simple interface for datasets (custom ParquetFileFormat)", {
```

Comment:
Out of curiosity, what does this test? How does it verify that we're obeying the reader options (and not just ignoring them)?

Reply:
Currently it only tests that nothing breaks. I'll amend the test to check that the schema includes the expected dictionary column